standard/format_table.lua

```diff
@@ -0,0 +1,91 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Format/Table
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Logic = require('Module:Logic')
+local Table = require('Module:Table')
+
+local TableFormatter = {}
+
+function TableFormatter.toLuaCode(inputTable)
+	if type(inputTable) ~= 'table' then
+		error('TableFormatter.toLuaCode needs a table as input')
+	end
+
+	if not next(tbl) then
```

Comment:
```suggestion
	if not next(inputTable) then
```
If you want it to work, though may be better to use `Table.isEmpty()` to not break on loadData